spines/bullet_spine.cpp

```diff
@@ -182,11 +192,11 @@ int main(const char* argv0, const CommandLineArguments& args) {
   BulletInterface::Parameters bullet_params(Dictionary{});
   bullet_params.argv0 = argv0;
   bullet_params.dt = 1.0 / args.spine_frequency;
-  bullet_params.floor = !args.space;
   bullet_params.gravity = !args.space;
   bullet_params.gui = args.show;
   bullet_params.position_base_in_world = Eigen::Vector3d(0., 0., base_altitude);
   bullet_params.urdf_path = "external/upkie_description/urdf/upkie.urdf";
+  bullet_params.extra_urdf_paths = args.extra_urdf_paths;
```

Comment:
PS: I am aware the "extra" is already merged in Vulp and I don't like revisiting that decision just because more brain cycles are available now to think about it :sweat_smile: So if you don't feel this is an improvement worth the changes we can skip it.